### PR TITLE
Automated cherry pick of #4342: Fix: etcd Authentication links.

### PR DESCRIPTION
**Pull request by lxpollitt**
Cherry pick of #4342 on release-v3.17.

#4342: Fix: etcd Authentication links.

### CODE REVIEW

**Review by bcreane — APPROVED** [review]
/lgtm

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Pull request merged**
